srcpkgs/jamulus/template

```diff
@@ -12,4 +12,4 @@ maintainer="astralchan <astral@astralchan.xyz>"
 license="GPL-2.0-or-later"
 homepage="https://jamulus.io"
 distfiles="https://github.com/corrados/jamulus/archive/${_version}.tar.gz"
```

Comment:
and remove $_version
```suggestion
distfiles="https://github.com/corrados/jamulus/archive/r${version//./_}.tar.gz"
```

Reply:
Why remove `$_version`?

Reply:
it's only used in one place

Reply:
It helps me mentally manage to expand it - even if it's just in one place. It helps me maintain it.

Reply:
It's a simple expression and it's only used in one place, when someone not familiar with the template reads it, it's not immediately obvious what $_version is, but it would be obvious if the expression was directly in $distfiles.
I am not sure how having separate $_version makes this more maintainable for you, could you explain that?

Reply:
I like evaluating expressions then substituting them. It makes it simpler for me to maintain.